adodbapi/README.md

```diff
@@ -0,0 +1,122 @@
+<!-- markdownlint-disable MD003 MD031 -->
+# adodbapi
+
+A Python [DB-API 2.0 (PEP 249)](https://peps.python.org/pep-0249/) module that makes it easy to use Microsoft ADO for connecting with databases and other data sources using CPython.
+
+Home page: <https://github.com/mhammond/pywin32/tree/main/adodbapi#readme>
+
+Features
+--------
+
+* 100% [DB-API 2.0 (PEP 249)](https://peps.python.org/pep-0249/) compliant (including most extensions and recommendations).
+* Includes [PyUnit](https://wiki.python.org/moin/PyUnit) testcases that describe how to use the module.
+* Fully implemented in Python. -- runs in current versions of Python 3.
+* Licensed under the LGPL license, which means that it can be used freely even in commercial programs subject to certain restrictions.
+* The user can choose between paramstyles: `qmark`, `named`, `format`, `pyformat` or `dynamic`
+* Supports data retrieval by column name e.g.:
+  ```python
+  for row in myCurser.execute("select name,age from students"):
+      print("Student", row.name, "is", row.age, "years old.")
+  ```
+* Supports user-definable system-to-Python data conversion functions (selected by ADO data type, or by column)
+
+Prerequisites
+-------------
+
+* Python 3.7 or higher
+
+Installation
+------------
+
+* (CPython on Windows): Install [pywin32](https://pypi.org/project/pywin32/) (`pip install pywin32`) which includes adodbapi.
+* (IronPython on Windows): Refer to <http://sf.net/projects/adodbapi> instead.
```

Comment:
Is it worth mentioning that the last version of pywin32 supporting IronPython is 306 ?